src/components/navigation/document-tab-content.tsx

```diff
@@ -39,6 +39,7 @@ export const DocumentTabContent: React.FC<IProps> = ({ tabSpec }) => {
   };
 
   const sectionClass = referenceDocument?.type === "learningLog" ? "learning-log" : "";
+  const selectedSection = tabSpec.tab==="supports"? ENavTabSectionType.kTeacherSupports :undefined;
```

Comment:
Spacing:
`const selectedSection = tabSpec.tab==="supports" ? ENavTabSectionType.kTeacherSupports : undefined;`

Reply:
IIRC, without specifically setting `selectedSection` here, the last tab the user had open will be remembered. I think it would be less disruptive to only override the default of opening to the tab the user last had open when there are new teacher supports available, i.e. something like:
```
  const selectedSection = (tabSpec.tab === "supports") && supports.hasNewTeacherSupports(user.lastSupportViewTimestamp)
          ? ENavTabSectionType.kTeacherSupports
          : undefined;
```